sql/core/src/test/scala/org/apache/spark/sql/StatisticsCollectionSuite.scala

```diff
@@ -57,7 +57,7 @@ class StatisticsCollectionSuite extends StatisticsCollectionTestBase with Shared
       }
 
       assert(sizes.size === 1, s"number of Join nodes is wrong:\n ${df.queryExecution}")
-      assert(sizes.head === BigInt(128),
+      assert(sizes.head === BigInt(129),
```

Comment:
Before this PR. It is 128 * 1 = 128, After this PR. It is 128 + 1 = 129. Please see:
https://github.com/apache/spark/blob/082d49aa810957a2cd6b4f4c06919e59fecca3bd/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/statsEstimation/SizeInBytesOnlyStatsPlanVisitor.scala#L133-L136